runtime/j9vm/j9memcategories.c

```diff
@@ -37,8 +37,13 @@
  * - The first category is the root, all categories in the tree must be reachable from that.
  */
 
+#if defined(OPENJ9_BUILD)
+RUNTIME_INFO = "RUNTIME"
+#else
+RUNTIME_INFO = "JRE"
+#endif
```

Comment:
Suggest move these macros into a common place like `j9.h` to avoid code duplication.